Configuration/Geometry/README.md

```diff
@@ -51,6 +52,6 @@ Several detector combinations have been generated:
 * D50 = T15+C9+M4+I11+O4+F2
 * D51 = T17+C9+M4+I10+O4+F2
 * D52 = T18+C9+M4+I10+O4+F2
-* D53 = T15+C9+M4+I12+O4+F2
```

Comment:
The D53 workflow is used for testing a new MTD geometry. This PR should not remove it. Please fix this throughout the PR.

Reply:
This should be fixed at commit cb085bb. Please confirm @kpedro88